clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/ManifestController.java

```diff
@@ -52,15 +53,24 @@ public ManifestController(List<ManifestProvider> manifestProviders, RequestQueue
   @PreAuthorize("hasPermission(#account, 'ACCOUNT', 'READ')")
   @PostAuthorize("hasPermission(returnObject?.moniker?.app, 'APPLICATION', 'READ')")
   @RequestMapping(value = "/{account:.+}/_/{name:.+}", method = RequestMethod.GET)
-  Manifest getForAccountAndName(@PathVariable String account, @PathVariable String name) {
-    return getForAccountLocationAndName(account, "", name);
+  Manifest getForAccountAndName(
+      @PathVariable String account,
+      @PathVariable String name,
+      @RequestParam(value = "includeEvents", required = false, defaultValue = "true")
+          boolean includeEvents) {
+    return getForAccountLocationAndName(account, "", name, includeEvents);
   }
 
   @PreAuthorize("hasPermission(#account, 'ACCOUNT', 'READ')")
   @PostAuthorize("hasPermission(returnObject?.moniker?.app, 'APPLICATION', 'READ')")
   @RequestMapping(value = "/{account:.+}/{location:.+}/{name:.+}", method = RequestMethod.GET)
   Manifest getForAccountLocationAndName(
-      @PathVariable String account, @PathVariable String location, @PathVariable String name) {
+      @PathVariable String account,
+      @PathVariable String location,
+      @PathVariable String name,
+      @RequestParam(value = "includeEvents", required = false, defaultValue = "true")
+          boolean includeEvents) {
```

Comment:
Looks like you forget to pass this parameter to the call to `getManifest` below?

Reply:
Good catch, looks like I messed this up when rebasing some of my changes